release/ray_release/tests/test_db_reporter.py

```diff
@@ -0,0 +1,57 @@
+from ray_release.reporter.db import DBReporter
+
+
+def test_compute_stack_pattern():
+    assert (
+        (DBReporter()).compute_crash_pattern(
+            """
+haha
+Traceback (most recent call last):
+    File "/tmp/something", line 584
+Exception: yaya45
+hehe
+"""
```

Comment:
nit: either move this to top level (out of the function), or use a `"\n".join([<string list>])`, so that the indentation looks a bit normal?